pom.xml

```diff
@@ -177,7 +178,12 @@
                 <groupId>jakarta.validation</groupId>
                 <artifactId>jakarta.validation-api</artifactId>
                 <version>${version.jakarta-validation}</version>
-                <scope>provided</scope>
+            </dependency>
+
+            <dependency>
+                <groupId>jakarta.json.bind</groupId>
+                <artifactId>jakarta.json.bind-api</artifactId>
+                <version>${version.jsonb}</version>
```

Comment:
We can remove this now right ?

Reply:
Yes, of course. Sorry. I just had seen that there are other modules that also depend on json-b, but they inherit the version from the `smallrye-parent-pom`, so it was wrong anyway.